packages/-serializer-encapsulation-test-app/tests/integration/push-payload-test.js

```diff
@@ -90,7 +90,7 @@ module('integration/push-payload - pushPayload method forwards to Serializer#pus
 
     const store = this.owner.lookup('service:store');
 
-    assert.expectAssertion(() => {
+    assert.throws(() => {
```

Comment:
Same question: why does this one become `assert.throws`, is the underlying thing a `throw new Error()` vs `assert(...)`?

Reply:
it's an assert but it throws like a normal error